apps/site/src/_pages/docs/5_faq.mdx

```diff
@@ -117,13 +123,25 @@ To save data beyond the session, blocks should make use of the [operations defin
 
 </FAQ>
 
-<FAQ question="How are Block Protocol blocks different to Web Components?">
+<FAQ question="What are the security considerations of blocks?">
 
-The Block Protocol standardizes how data requests are made between a block and an embedding application.
+Similar to most programming language package managers, because anybody can publish executable code (in the form of a block), it's important you trust the block you're executing.
 
-Web Components (or custom elements) are one way of implementing Block Protocol blocks.
+In most applications, the users inserting new blocks generally aren't developers, which means 
```

Comment:
Something missing here

Reply:
Addressed in https://github.com/blockprotocol/blockprotocol/pull/903/commits/fd6d0e92796b146a4674e7dd24393b671611d6ea